docs/concepts/metrics.md

```diff
@@ -0,0 +1,85 @@
+---
+id: metrics
+title: Temporal Metrics
```

Comment:
```suggestion
title: Worker Entity metrics reference
```
Is this more accurate?

Reply:
Kinda. It's more like Worker & SDK metrics. I'll go for that